x/ccv/provider/types/key_assignment.go

```diff
@@ -9,6 +9,18 @@ import (
 	ccvtypes "github.com/cosmos/interchain-security/v2/x/ccv/types"
 )
 
+// A validator's consensus address on the provider chain.
+type ProviderConsAddress struct {
```

Comment:
Prev these types were generated in .pb files, now we just redefine the structs here without the marshal/unmarshal methods. Ie. to persist these types you need to call `ToSdkConsAddr()`